src/NuGet.CommandLine/Program.cs

```diff
@@ -86,10 +88,31 @@ public int Main(string[] args)
                         project,
                         sources.Values.Select(s => new PackageSource(s)),
                         packagesDir);
+                    if (parallel.HasValue())
```

Comment:
Could this work like a switch where it picks a default value (4 \ 8) if it doesn't have one? Would be nice to go
`restore --parallel` and have it use sensible defaults.